api/src/main/java/net/countercraft/movecraft/CruiseDirection.java

```diff
@@ -53,5 +53,33 @@ else if(direction.getOppositeFace() == BlockFace.DOWN)
         else
             return NONE;
     }
+
+    public CruiseDirection getOpposite() {
+        return switch (this) {
+            case NORTH -> SOUTH;
+            case SOUTH -> NORTH;
+            case EAST -> WEST;
+            case WEST -> EAST;
+            case UP -> DOWN;
+            case DOWN -> UP;
+            case NONE -> NONE;
+        };
+    }
+
+    public CruiseDirection getClockwiseRotation() {
+        return switch (this) {
+            case NORTH -> EAST;
+            case SOUTH -> WEST;
+            case EAST -> SOUTH;
+            case WEST -> NORTH;
+            case UP -> UP;
+            case DOWN -> DOWN;
+            case NONE -> NONE;
+        };
+    }
+
+    public CruiseDirection getAnticlockwiseRotation() {
+        return this.getClockwiseRotation().getOpposite();
+    }
```

Comment:
Rather than use two functions, let's combine them into one:
```java
public CruiseDirection getRotated(MovecraftRotation rotation) {
    return switch(rotation) {
        case CLOCKWISE -> switch (this) {
            case NORTH -> EAST;
            case SOUTH -> WEST;
            case EAST -> SOUTH;
            case WEST -> NORTH;
            case UP -> UP;
            case DOWN -> DOWN;
            case NONE -> NONE;
        };
        case ANTICLOCKWISE -> getRotated(MovecraftRotation.CLOCKWISE).getOpposite();
        case NONE -> this;
    };
}
```